pkg/swagger/swagger.go

```diff
@@ -71,12 +71,16 @@ func Run(outputDir string) error {
 
 	populateExamples(s.Paths)
 
-	err := define(s.Definitions, "github.com/Azure/ARO-RP/pkg/api/v20191231preview", "OpenShiftClusterList", "OpenShiftClusterCredentials")
```

Comment:
are you intentionally removing this line?

Reply:
it looks like it was moved below and changed